### PR TITLE
Fixes #10020: Document usage of "file ensure keys->values present"

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/10020

### CODE REVIEW

File: tree/30_generic_methods/file_ensure_keys_values.cf [hunk 1]
```diff
@@ -18,6 +18,57 @@
...
+#    "key2": "value2"
+# }
+# ```
+#
+# With the following policy:
+#
+# ```
+# # Define the `content` variable in the `configuration` prefix from the json file
+# variable_dict_from_file("configuration", "content", "/tmp/data.json")
+# # Expand yout template
```
Comment (ncharles) [inline tree/30_generic_methods/file_ensure_keys_values.cf]: typo: your

**Comment by amousset** [comment]
Commit modified

**Comment by ncharles** [comment]
ha, you fixed the typo while I was commenting :)

**Comment by ncharles** [comment]
<3

**Pull request merged**